db/migrate/20160618224927_merge_user_tables.rb

```diff
@@ -0,0 +1,6 @@
+class MergeUserTables < ActiveRecord::Migration
```

Comment:
I'm a little confused by this migration because the User table already has user_id as integer as well as name as string from when I created and merged the database structure =/